packages/d-ser-t-service/package.json

```diff
@@ -1,52 +1,60 @@
-  "version": "1.1.3-alpha.2",
-  "description": "Dynamic Sentence Error Rate Testing: A Package for testing the CRIS speech-to-text model, quantifying the quality of the model with respect to its Word Error Rate",
-  "license": "MIT",
-  "author": "Yomi (@jolly59), Katie(@KatieProchilo), Nick (@nhuntwalker) ",
-  "main": "lib/main.js",
-  "types": "lib/main.d.ts",
-  "homepage": "https://github.com/Joll59/d-ser-t/blob/master/packages/d-ser-t-service/README.md",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Joll59/d-ser-t.git"
-  },
-  "bugs": {
-    "url": "https://github.com/Joll59/d-ser-t/issues"
-  },
-  "directories": {
-    "lib": "lib"
-  },
-  "files": [
-    "/lib"
-  ],
-  "keywords": [
-    "d-ser-t",
-    "d-ser-t-cli",
-    "speech-recognition",
-    "sentence-error-rate",
-    "word-error-rate",
-    "testing",
-    "batch",
-    "custom-speech-model"
-  ],
-  "dependencies": {
-    "colors": "^1.3.3",
-    "diff": "^4.0.1",
-    "microsoft-cognitiveservices-speech-sdk": "1.5.0",
-    "tslib": "^1.10.0",
-    "word-error-rate": "0.0.7"
-  },
-  "devDependencies": {
-    "@types/diff": "^4.0.2",
-    "@types/jest": "^24.0.15",
-    "jest": "^24.8.0",
-    "ts-jest": "^24.0.2"
-  },
-  "scripts": {
-    "build": "tsc --build",
-    "clean": "del-cli lib tsconfig.tsbuildinfo node_modules",
-    "test": "jest --colors --verbose --config jest.json --rootDir . --passWithNoTests"
-  },
-  "gitHead": "d28b5afec325253aaea5dcaee00cbfef083e4655"
-}
+    "name": "d-ser-t-service",
+    "version": "1.1.3-alpha.2",
+    "description": "Dynamic Sentence Error Rate Testing: A Package for testing the CRIS speech-to-text model, quantifying the quality of the model with respect to its Word Error Rate",
+    "license": "MIT",
+    "author": "Yomi (@jolly59), Katie(@KatieProchilo), Nick (@nhuntwalker) ",
+    "main": "lib/main.js",
+    "types": "lib/main.d.ts",
+    "homepage": "https://github.com/Joll59/d-ser-t/blob/master/packages/d-ser-t-service/README.md",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Joll59/d-ser-t.git"
+    },
+    "bugs": {
+        "url": "https://github.com/Joll59/d-ser-t/issues"
+    },
+    "directories": {
+        "lib": "lib"
+    },
+    "files": [
+        "/lib"
+    ],
+    "keywords": [
+        "d-ser-t",
+        "d-ser-t-cli",
+        "speech-recognition",
+        "sentence-error-rate",
+        "word-error-rate",
+        "testing",
+        "batch",
+        "custom-speech-model"
+    ],
+    "dependencies": {
+        "colors": "^1.3.3",
+        "diff": "^4.0.1",
+        "microsoft-cognitiveservices-speech-sdk": "1.5.0",
+        "tslib": "^1.10.0",
+        "word-error-rate": "0.0.7"
+    },
+    "devDependencies": {
+        "@types/diff": "^4.0.2",
+        "@types/jest": "^24.0.15",
+        "jest": "^24.8.0",
+        "prettier": "1.18.2",
+        "ts-jest": "^24.0.2"
+    },
+    "scripts": {
+        "build": "tsc --build && npm run check",
```

Comment:
As we are enforcing these style rules, the build step should fix before building.

Reply:
Resolved with https://github.com/Joll59/d-ser-t/commit/4ce1909bffe64446bb8e9d9281085e8a4c383140.

Reply:
Resolved with https://github.com/Joll59/d-ser-t/commit/4ce1909bffe64446bb8e9d9281085e8a4c383140.